open-telemetry-addon/deploy/resources/addonmanager.role.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: addon-manager
```

Comment:
the name needs to be specific for prometheus

Reply:
how about `otel-addon-manager`?